tox.ini

```diff
@@ -93,7 +93,7 @@ allowlist_externals=
     bash
 commands =
     bash -c 'if [[ "{env:GITHUB_ACTIONS:}" == "true" ]]; then playwright install --with-deps; else playwright install; fi'
-    pytest -m "a11y" {posargs}
+    pytest --tracing=retain-on-failure -m "a11y" {posargs}
```

Comment:
Most guides that I found on uploading [traces with CI](https://playwright.dev/docs/trace-viewer#tracing-on-ci), configure Playwright to record a trace "on-first-retry," but I don't think we do retries. I guess we haven't had a need for it, which means our tests are not flaky, which is great. 

But [another part of the Playwright documentation on setting up CI](https://playwright.dev/python/docs/ci-intro#setting-up-github-actions) uses `--tracing=retain-on-failure`, so I think this is fine.